docs/index.html

```diff
@@ -80,120 +80,122 @@ <h2>What is the purpose?</h2>
-        moved: 0
-      },
-      {
-        quadrant: 1,
-        ring: 0,
-        label: "SendGrid",
-        website: "https://sendgrid.com",
-        active: true,
-        moved: 0
+        moved: 0,
+        description: `
+        <p>Locust is an open source load testing framework written in Python.</p>
+        <p>We use Locust to ensure Small Improvements can handle higher than expected traffic and to test potential bottlenecks in our application.</p>
+        `
       },
       {
         quadrant: 1,
         ring: 0,
         label: "Docker",
         website: "https://www.docker.com",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Docker is a container virtualization service that packages an application and its dependencies in a container that can run on any Linux</p>
+        <p>We use Docker to perform repeatable builds of our application and for local development in order to spin up mock versions of cloud services.</p>
+        `
       },
       {
         quadrant: 1,
         ring: 0,
         label: "TrackJS",
         website: "https://trackjs.com",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>TrackJS is an error tracking platform.</p>
+        <p>We use TrackJS to monitor our application for errors experienced by clients.</p>
+        `
       },
       {
         quadrant: 1,
         ring: 0,
         label: "Google Analytics",
         website: "https://analytics.google.com",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Google Analytics is a web analytics platform.</p>
+        <p>We use Google Analytics to monitor usage of our application (all data is anonymized). This data helps us understand where we can add more value to the platform in the future.</p>
+        `
       },
       {
         quadrant: 2,
         ring: 0,
         label: "App Engine",
-        website: "",
+        website: "https://cloud.google.com/appengine",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Google App Engine is a Platform as a Service (PaaS) cloud computing platform.</p>
+        <p>The majority of our cloud infrastructure runs on App Engine. This allows us to focus on building our application instead of worrying about servers and networks.</p>
+        `
       },
       {
         quadrant: 2,
         ring: 0,
-        label: "Datastore",
+        label: "Google Datastore",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Google Datastore is a NoSQL database.</p>
+        <p>Datastore goes hand-in-hand with App Engine and hosts our main database. The datastore scales very well and is able to handle a very high load.
+          As with the App Engine, Datastore is fully managed so we don't have to worry about bootstrapping servers and maintainence released.</p>
```

Comment:
Something's off at the end. Did you mean this?

```suggestion
          As with the App Engine, Datastore is fully managed so we don't have to worry about bootstrapping servers and maintaining releases.</p>
```